microsoft-edge/webview2/how-to/webdriver.md

```diff
@@ -229,6 +229,11 @@ For more info about the `DebuggerAddress` property of the `EdgeOptions` object,
 
 Congratulations!  You've successfully automated a WebView2 project and driven WebView2 by using Selenium and Microsoft Edge WebDriver, by attaching Microsoft Edge WebDriver to a running WebView2 app.
 
+### Note for using UWP projects and debugging
+Because of the way that UWP projects are launched the WebDriver is not able to automatically configure the debug settings on launch. You will need to use the registry and enable debugging for all WebView2 process on startup.
+WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS environment variable (and corresponding regkey) that are documented in our https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.818.41 Microsoft Docs page. The browser arguments env var / regkey lets you add to the command line arguments that will be passed to the WebView2 Runtime browser process when it is started.
```

Comment:
```suggestion
For information about the WEBVIEW2_ADDITIONAL_BROWSER_ARGUMENTS environment variable (and the corresponding registry key), see [CreateCoreWebView2EnvironmentWithOptions](/microsoft-edge/webview2/reference/win32/webview2-idl?#createcorewebview2environmentwithoptions) in _Globals_. The browser arguments `env var / regkey` let you add to the command line arguments that will be passed to the WebView2 Runtime browser process when it is started.
```
clickable link:
https://learn.microsoft.com/en-us/microsoft-edge/webview2/reference/win32/webview2-idl?view=webview2-1.0.818.41
= truncated link per Docs build:
/microsoft-edge/webview2/reference/win32/webview2-idl

Reply:
While it's true you can configure UserDataFolder, this is being used in reference to enable debugging.  So the information about the UDF could be confusing.

Reply:
I edited my comment/suggestion tagging ( ... > Edit ) above to remove:
For example, you can add a value for `userDataFolder`.
@DavidShoe

Reply:
Committing based on Michael's request.